project.clj

```diff
@@ -18,4 +21,34 @@
               :dependencies [[criterium "0.4.4"]
                              [org.clojure/tools.cli "0.3.3"]
                              [org.clojure/test.check "0.9.0"]
-                             [com.infolace/excel-templates "0.3.3"]]}})
+                             [com.infolace/excel-templates "0.3.3"]]}
+             :dev {:dependencies [[binaryage/devtools "0.8.2"]
+                                  [figwheel-sidecar "0.5.8"]
+                                  [com.cemerick/piggieback "0.2.1"]
+                                  [org.clojure/test.check "0.9.0"]]
+                   :source-paths ["src" "dev"]
+                   ;; need to add dev source path here to get user.clj loaded
+                   ;; TODO check whether this still makes sense,
+                   ;; workaround for cider cljs REPL
+                   #_:figwheel #_{:nrepl-port 7888
```

Comment:
I am not sure how to set it up best. The cljs toolchain has gotton way better, but figwheel has changed its integration and cider is not integrating with the cljs REPL well, so I tried to work around this. But I will remove it again, I started figwheel manually from the clj REPL in cider and that worked ok (not perfect though, as cider does not recognize the cljs repl as belonging to the clj REPL then). Using sidecar is the official way though, so I would just remove the comment.